### ISSUE TITLE
URLs outdated

### ISSUE THREAD

**Issue by aurelijusb** [versioneye/versioneye_sbt_plugin#13]
When using:

``` scala
addSbtPlugin("com.versioneye" % "sbt-versioneye-plugin" % "0.2.0")
```

And then

```
> versioneye:createProject
[info] .
[info] Starting to upload dependencies to https://www.versioneye.com. This can take a couple seconds ... 
[info] .
[trace] Stack trace suppressed: run last versioneye:createProject for the full output.
[error] (versioneye:createProject) Status Code: HTTP/1.1 400 Bad Request
```

Does www.versioneye.com changes its API?

**Comment by reiz** [comment]
@aurelijusb No. The API location is unchanged since years. It is here: https://www.versioneye.com/api/. @mp911de Can you re produce this error message?

**Comment by mp911de** [comment]
I'll take a look into it. Another thing came just to my mind. @reiz do you use requestId's? If so, great. If not, that would be a great addition so we could:
1. Provide a feedback to users and they could report the Id in tickets
2. RequestIds provide a diagnostic context and you can track that data on the server side (with some retention) so it's possible to see what happened during the request.

**Comment by reiz** [comment]
Hi @mp911de. Currently we don't use request IDs. But I like the idea. The logging in VersionEye needs to be refactored anyway.

**Comment by mp911de** [comment]
Ping me if you like to hear more. I could head over to your place and have a chat with you about that.

**Comment by reiz** [comment]
OK. Currently the problem is that we don't have centralised logging. As soon that is fixed I would like to chat with you about request IDs :)
